packaging/windows/product.wxs.template

```diff
@@ -92,24 +92,6 @@
             Before="AddUserToHypervAdminGroup"
             Sequence="execute"/>
         <CustomAction Id="AddUserToHypervAdminGroup" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Impersonate="no" Return="ignore" />
-        <SetProperty Action="CACreateSMBShare"
-            Id="CreateSMBShare"
-            Value="&quot;[POWERSHELLEXE]&quot; -NonInteractive -ExecutionPolicy Bypass -NoProfile -Command &quot;New-SmbShare -Name '[SHAREDDIRNAME]' -Path '[USERFOLDER]' -FullAccess '[LogonUser]'&quot;"

```

Comment:
I guess you should also be removing 
```
        <Property Id="USERFOLDER">
            <DirectorySearch Id="userProfileSearch" Depth="0" Path="[%USERPROFILE]" />
        </Property>

        <Property Id="SHAREDDIRNAME" Secure="yes">crc-dir0</Property>
```
and possibly other properties? (powershell, logonuser, …)

Reply:
we need to keep powershell and logonuser these are used by the installHyperv custom action